### ISSUE TITLE
Enhancement: Inline `yarn.lock` modification

### ISSUE THREAD

**Issue by scinos** [atlassian/yarn-deduplicate#6]
*Originally reported by* **Ruslan Arkhipau (Bitbucket: [rarkhipau-at-atlassian](https://bitbucket.org/rarkhipau-at-atlassian), GitHub: Unknown)**

----------------------------------------

Right now I'm running this sequence of commands for "inline" modification:

```
yarn-tools fix-duplicates yarn.lock "some-dependency" > yarn.fixed.lock
rm yarn.copy.lock
mv yarn.fixed.lock yarn.lock
```

I see no major obstacles for that being the default behavior of `yarn-tools`

----------------------------------------
- Bitbucket: https://bitbucket.org/atlassian/yarn-tools/issue/6

**Comment by scinos** [comment]
Fixed in `1.0-dev`. I'll close this issue once the patch is released as part of version 1.0

**Comment by scinos** [comment]
Solved in 1.0.0
